utils/guardrails/guard.py

```diff
@@ -189,7 +182,7 @@ def evaluate(
     parser = argparse.ArgumentParser()
     parser.add_argument('--message', type=str, help='message to check')
     parser.add_argument('--role', type=str, help='role of the message')
-    parser.add_argument('--api', default='sambaverse', type=str, help='sambaverse or sambastudio')
+    parser.add_argument('--api', default='sambastudio', type=str, help='api to use')
```

Comment:
Just to confirm, shouldn't the default here be sncloud?

Reply:
no llamaguard is not available yet in sncloud